learn/ai_powered_search/multimodal.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Image search with user-provided embeddings — Meilisearch documentation
+description: This article shows you the main steps for searching through images with no associated metadata
+---
+
+# Image search with user-provided embeddings
+
+This article shows you the main steps for performing multimodal searches where you can use text to search through a database of images with no associated metadata.
+
+## Requirements
+
+- A database of images
+- A Meilisearch project
+- An embedding generation tool you can install locally
+
+## Configure your local embedding generation pipeline
+
+First, set up a system that sends your images to your chosen embedding generation tool.
+
+The exact procedure depends heavily on your specific setup, but should include these main steps:
+
+1. Choose a tool you can run locally
```

Comment:
We could give a few precisions about the "tool" (it's a model than can at least do image to text embedding). This tool could also be a third party that does not run locally but over the network